store/sql_post_store.go

```diff
@@ -642,6 +642,7 @@ func (s SqlPostStore) Search(teamId string, userId string, params *model.SearchP
 				Posts
 			WHERE
 				DeleteAt = 0
+				AND Type NOT LIKE '` + model.POST_SYSTEM_MESSAGE_PREFIX + `%'
```

Comment:
I can run some analysis on it but I don't think it should be that slow since the field it's checking against isn't very long. Instead of using a LIKE clause I can do post-processing on the rows returned by the query as well, though I'm not sure if that would be faster or slower.

I'll let you know what I find